DependencyInjection/CmfSeoExtension.php

```diff
@@ -27,6 +26,9 @@
  */
 class CmfSeoExtension extends Extension
 {
+    const CLASS_SEO_METADATA_PHPCR = 'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SeoMetadata';
+    const CLASS_SEO_METADATA_ORM = 'Symfony\Cmf\Bundle\SeoBundle\Document\SeoMetadata';
```

Comment:
i think even just loading the form would fail as it would try to create
the data_class. not sure though.